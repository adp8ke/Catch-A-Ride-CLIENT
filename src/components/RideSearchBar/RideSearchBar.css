.searchbar {
  background:rgba(26,32,54,.47);
  border: 1px solid white;
  border-radius: 5px;
  width: 80%;
  margin: 0 auto;
  padding-bottom: 21px;
  color: white;
}

@media only screen and (min-width: 700px) {
  .searchbar {
    /* min-width: 554px; */
    width: 40%;
  }
}