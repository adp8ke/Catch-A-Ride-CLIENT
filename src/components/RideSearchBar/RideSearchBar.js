import React, { Component } from 'react';
import SearchBox from '../SearchBox/SearchBox';
import './RideSearchBar.css';

export default class RideSearchBar extends Component {
    render() {
        return (
            <div className='searchbar'>
                <h3>Search For Rides</h3>
                <SearchBox />
            </div>
        )
    }
}