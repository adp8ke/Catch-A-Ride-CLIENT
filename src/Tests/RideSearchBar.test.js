import React from 'react';
import ReactDOM from 'react-dom';
import {MemoryRouter} from 'react-router-dom';
import RideSearchBar from '../components/RideSearchBar/RideSearchBar';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<MemoryRouter><RideSearchBar /></MemoryRouter>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
  const tree = renderer
    .create(<MemoryRouter><RideSearchBar /></MemoryRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});