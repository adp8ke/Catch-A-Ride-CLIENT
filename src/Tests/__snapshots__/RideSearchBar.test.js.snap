// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="searchbar "
>
  <h3>
    Search For Rides
  </h3>
  <form
    className="search-form"
    onSubmit={[Function]}
  >
    <label
      className="search-start"
      htmlFor="start-input"
    >
      Starting: 
    </label>
    <input
      id="start-input"
      onChange={[Function]}
      placeholder="Enter City Name"
      value=""
    />
    <br />
    <label
      className="search-dest"
      htmlFor="dest-input"
    >
      Destination: 
    </label>
    <input
      id="dest-input"
      onChange={[Function]}
      placeholder="Enter City Name"
      value=""
    />
    <br />
    <button
      className="search-submit"
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
